### PR TITLE
chore: Deprecate kyma-cli v2

### PR DESCRIPTION
**Pull request by pPrecel**
**Description**

Changes proposed in this pull request:

- add a deprecation note at the beginning of the main `README.md` file
- add adeprecation note at the beginning of the kyma-cli `main.go` file

**Related issue(s)**

- https://github.com/kyma-project/cli/issues/2051

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 # Kyma CLI
 
+> [!WARNING]
```
Comment (pPrecel) [inline README.md:5]: It's based on [this](https://github.com/orgs/community/discussions/16925) article

File: README.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
...
+> All current commands are deprecated and new `v3` commands will be developed within `alpha` command group.
```
Comment (kwiatekus) [inline README.md:6]:
```suggestion
> Version `v2` of kyma CLI is deprecated. This implies that all commands available within version `v2` are also deprecated. We started designing `v3` commands that will be first released within `alpha` command group.
```

File: cmd/main.go [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
 import (
+	"fmt"
 	"os"
 
 	"github.com/kyma-project/cli/cmd/kyma"
 	"github.com/kyma-project/cli/internal/cli"
 )
 
 func main() {
+	fmt.Print("WARNING: All current commands are deprecated and new v3 commands will be developed within alpha command group.\n\n")
```
Comment (kwiatekus) [inline cmd/main.go:12]:
```suggestion
	fmt.Print("WARNING: All commands within v2 are deprecated. We are designing v3 of kyma CLI  with a new set of commands that will be first released  within alpha command group.\n\n")
```

**Review by kwiatekus — DISMISSED** [review]

File: README.md [hunk 4]
```diff
@@ -2,6 +2,10 @@
 
 # Kyma CLI
 
+> [!WARNING]
+> Version `v2` of kyma CLI is deprecated. This implies that all commands available within version `v2` are also deprecated. We started designing `v3` commands that will be first released within `alpha` command group.
```
Comment (grego952) [inline README.md:6]:
```suggestion
> The Kyma CLI version `v2`, with all commands available within this version, is deprecated. We've started designing the `v3` commands that will be first released within the `alpha` command group.
```

**Review by grego952 — APPROVED** [review]

**Pull request merged**